src/main/groovy/life/qbic/samplestatus/reporter/commands/ReportSinceInstant.groovy

```diff
@@ -82,6 +82,7 @@ class ReportSinceInstant implements Runnable {
       updatedSamples.stream()
               .filter(Result::isOk)
               .map(Result::getValue)
+              .sorted((SampleUpdate su1, SampleUpdate su2) -> su1.getModificationDate().compareTo(su2.getModificationDate()))
```

Comment:
what does this do and why was it necessary? did not find information about it in the PR

Reply:
ah, just sorts by modification date?

Reply:
yes